2020-01-Winter/4_trees/Kth_Smallest_Element_in_BST/recursion_KthSmallest_in_BST.py

```diff
@@ -0,0 +1,43 @@
+class TreeNode:
+     def __init__(self, x):
+         self.val = x
+         self.left = None
+         self.right = None
+
+def findNode(node, count, result):
+  if result[0] != -1:
```

Comment:
Match source code file to README version.
```suggestion
  if result:
```